modules/backend/behaviors/ListController.php

```diff
@@ -473,8 +476,9 @@ public function listFilterExtendScopes($host)
 
     /**
      * Controller override: Extend supplied model
-     * @param Model $model
-     * @return Model
+     * @param \October\Rain\Database\Model|\October\Rain\Halcyon\Model $model
+     * @param string|null $definition
+     * @return \October\Rain\Database\Model|\October\Rain\Halcyon\Model
```

Comment:
@jacobdekeizer I don't think that the Lists widget actually supports Halcyon models given the deep integration with the Eloquent query builder